root/static/scripts/tests/release-editor/fields.js

```diff
@@ -183,3 +183,30 @@ fieldTest("tracks are set correctly when the cdtoc is changed", function (t, rel
     );
     t.ok(_.last(medium.tracks()).isDataTrack());
 });
+
+fieldTest("Tracks' time are changed correctly when inputting values in the medium tracklist editing form tab", function (t, release){
+    t.plan(8);
+
+    var medium = new fields.Medium({ tracks: [ {} ] }, release);
+    
+    var lengths = [
+        {5},
+        {69},
+        {174},
+        {6000},
+        {7400},
+        {10000},
+        {96900},
+        {3723494},
+    ];
+
+
+    t.equal(medium.tracks()[0].formattedLengthChange(lengths[0]), "0:05", );
+    t.equal(medium.tracks()[1].formattedLengthChange(lengths[1]), "1:09", );
```

Comment:
1. There is only one track in this medium, `tracks()[0]` should be used for every test;
2. `formattedLengthChanged` is missing its final `d`;
3. `formattedLengthChanged` does not return anything, it updates `formattedLength`;
4. a test description would be appreciated.

Here is an example to fix this test.
```javascript
medium.tracks()[0].formattedLengthChange(lengths[1]);
t.equal(medium.tracks()[1].formattedLength(), "1:09", "length " + lengths[1] + " is formatted as 1:09");
```
Plus, I suggest storing expected formatted lengths ("1:09" here) in an array too, so as to iterate tests rather than duplicating code lines.

Reply:
There was a mistake in the second line of my example, it should always be `tracks()[0]`.